waku/v2/node/waku_switch.nim

```diff
@@ -9,7 +9,7 @@ import
   chronos, chronicles,
   eth/keys,
   libp2p/crypto/crypto,
-  libp2p/protocols/pubsub/gossipsub,
+  libp2p/protocols/[pubsub/gossipsub, rendezvous],
```

Comment:
We generally have imports fully qualified and in separate lines (collective imports like this are difficult to "find and replace" at later stage). See [coding guidelines](https://hackmd.io/@vac/main/%2F5oIVOuQcSM2WKrnDLUPcpg#Importsexports)